operator/src/main/java/oracle/kubernetes/operator/helpers/ServiceHelper.java

```diff
@@ -99,10 +99,17 @@ public NextAction apply(Packet packet) {
       List<V1ServicePort> ports = new ArrayList<>();
       V1ServicePort servicePort = new V1ServicePort();
       servicePort.setPort(port);
+      servicePort.setName("weblogic");
       if (nodePort != null) {
         servicePort.setNodePort(nodePort);
       }
       ports.add(servicePort);
+
```

Comment:
Why are you exposing the node manager port here if we didn't before?

Reply:
Sorry, this was leftover code when we were trying to address the WLDF delay issue by exposing nm port that I forgot to remove.